### ISSUE TITLE
Request to update status - maybe on wiki page

### ISSUE THREAD

**Issue by riftware** [anidev/frc-driverstation#10]
Greetings folks, 
   I pulled this project because its stated goal is to accomplish exactly what I need.    However its unclear if some of the problems I'm encountering are due to the fact the functionality is not yet implemented or if its in fact new problems.   Can we get a state of the union put up somewhere?  Wiki would be fine.
1: Running on windows 64 the commandline indicates a dll issue that might just be a misleading message, not sure but it seems to indicate the dll is not correct win32 (I didn't want win32, I'm running 64bit java so I'm confused)

2: On linux, on a system where I can connect to the roborio's admin page the driverstation implementation shows no communication with the roborio.   Its unclear if its because 2015 isn't far enough along or if there is something actually wrong.  I've tried running as both a normal user and using sudo

3: While the ds saves the team number between runs the protocol version always reverts to 2009 - 2014 instead of remember 2015 once its been set.

The windows install is windows 7 64 bit with 64bit java 8 Sdk.  (java 32 bit may also be on the machine due to another program requiring it be around. but its not the one in the path or in the java home variable.

The linux machine is kubuntu/ubuntu 14.10 with java 8 sdk.

**Comment by LoadingPleaseWait** [comment]
Hi @riftware,
I'm also an Ubuntu user so I can answer questions 2 and 3. We are not done reverse engineering and implementing the new protocol yet so right now the project only works with cRIO robots. As for it not remembering which protocol to use, that feature has not been implemented either. If you want you could help us implement those things :smile:.
By the way, on a side note, you might want to upgrade to the newest version of Ubuntu, [Ubuntu 15.04 Vivid Vervet](https://wiki.ubuntu.com/VividVervet/ReleaseNotes), which came out about a week ago.

**Comment by riftware** [comment]
Yeah I just noticed that. My systems in the process of upgrading I actually
run  kubuntu so I usually don't get the upgraded exactly the same time as
ubuntu but it was available so its upgrading.  I will be looking at
contributing.  my  point on the original message was there maybe value in
putting on a wiki page on the GitHub the status.  I.e. indicating that 2015
doesn't connect yet would help a lot of people who download and play with
it I think.  Maybe that will be my first contribution if nobody  objects is
starting that off

**Comment by LoadingPleaseWait** [comment]
I agree. Having a wiki that says that among other things might be helpful.

**Comment by anidev** [comment]
Right now most of the indicators on the GUI aren't actually operational, so even if it is connected it won't display that. I've stopped working on most driver station features for now as the new protocol is being implemented. I'll get back to you on specific answers to your question and status updates later when I have more time. Thanks for offering to contribute! Most work now is going on in the protocol.

**Comment by anidev** [comment]
Having a wiki is a great idea, I'll start working on that when I get a chance. If anyone wants to contribute to that, feel free, I'm not going to get time for a week or two at least. In any case, @LoadingPleaseWait  is right about questions 2 and 3.

Regarding your first question, was there a particular error you were getting? As far as I know, on Windows it should be extracting 5 DLLs, which should include both 32bit and 64bit versions. I've only ever tested it on 32bit Windows. I can test it on 64bit Windows soon and see what happens, but were you getting an error or exception?
